models/src/main/java/com/vimeo/networking2/SeasonConnections.kt

```diff
@@ -15,4 +16,9 @@ data class SeasonConnections(
     @Json(name = "videos")
     val videos: Connection? = null
 
-)
+): Serializable {
```

Comment:
Formatting issue here, also noticed a few other DTOs with formatting issues, can you run the formatter on each DTO to be safe?

Reply:
Formatting updated 👍